src/components/ContinuationIn/ExtraproRegistration.vue

```diff
@@ -0,0 +1,331 @@
+              in the home jurisdiction.
+            </MessageBox>
+          </v-col>
+        </v-row>
+
+        <v-row
+          class="mt-8"
+          no-gutters
+        >
+          <v-col
+            cols="12"
+            sm="3"
+          >
+            <label>Confirmation</label>
+          </v-col>
+
+          <v-col
+            cols="12"
+            sm="9"
+          >
+            <v-checkbox
+              id="extrapro-registration-checkbox"
+              v-model="checkbox"
+              class="pt-0 mt-0"
+              hide-details
+              :label="checkboxLabel"
+            />
+          </v-col>
+        </v-row>
+      </template>
+
+      <template v-else>
+        <v-row
+          class="mt-2"
+          no-gutters
+        >
+          <v-col
+            cols="12"
+            sm="3"
+          >
+            <!-- empty column to line up with Business Information above -->
+          </v-col>
+
+          <v-col
+            cols="12"
+            sm="9"
+          >
+            <MessageBox color="red">
+              <strong>Important:</strong> This extraprovincial registration is historical.
+              Please contact BC Registries staff.
+            </MessageBox>
+
+            <RegistriesContactInfo class="mt-6" />
+          </v-col>
+        </v-row>
+      </template>
+    </template>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Vue, Watch } from 'vue-property-decorator'
+import { Action, Getter } from 'pinia-class'
+import { useStore } from '@/store/store'
+import { isEqual } from 'lodash'
+import { getName } from 'country-list'
+import { BusinessLookupResultIF, EmptyBusinessLookup } from '@/interfaces'
+import ExtraproBusinessLookup from './ExtraproBusinessLookup.vue'
+import MessageBox from '@/components/common/MessageBox.vue'
+import RegistriesContactInfo from '@/components/common/RegistriesContactInfo.vue'
+
+@Component({
+  components: {
+    ExtraproBusinessLookup,
+    MessageBox,
+    RegistriesContactInfo
+  }
+})
+export default class ExtraproRegistration extends Vue {
+  @Getter(useStore) getContinuationInBusinessInfo!: any
+  @Getter(useStore) getShowErrors!: boolean
+
+  @Action(useStore) setContinuationInBusinessInfo!: (x: any) => void
+
+  // Local properties
+  active = false
+  business = null // FUTURE: update BusinessLookupResultIF with new properties
+  checkbox = false
+
+  readonly checkboxLabel = 'I understand and acknowledge that the extraprovincial registration of this' +
+    ' business will be cancelled and made historical in the B.C. Registry once the continuation in' +
+    ' application has been submitted.'
+
+  /** Whether we have a looked-up business. */
+  get haveLookupBusiness (): boolean {
+    return !isEqual(this.business, EmptyBusinessLookup)
+  }
+
+  /** The identifier. */
+  get identifier (): string {
+    return this.business.identifier
+  }
+
+  /** The jurisdiction. */
```

Comment:
For now I'm calling the object `homeJurisdiction`, but I'm aware that COLIN calls this "foreign jurisdiction" (and also contains some extra properties).

This is working currently. We can easily change it later if needed. (Agile ftw!)